src/components/VencordSettings/UpdaterTab.tsx

```diff
@@ -197,6 +196,33 @@ function Updater() {
         repoPending
     };
 
+    return (
+        <>
+            <Forms.FormTitle tag="h5">Repo</Forms.FormTitle>
+            <Forms.FormText className="vc-text-selectable">
+                {repoPending
+                    ? repo
+                    : err
+                        ? "Failed to retrieve - check console"
+                        : (
+                            <Link href={repo}>
+                                {repo.split("/").slice(-2).join("/")}
+                            </Link>
+                        )}
+                {" "}(<HashLink hash={gitHash} repo={repo} disabled={repoPending} />)
+            </Forms.FormText>
+            <Forms.FormDivider className={Margins.top8 + " " + Margins.bottom8} />
+            <Forms.FormTitle tag="h5">Updates</Forms.FormTitle>
+
+            {isNewer ? <Newer {...commonProps} /> : <Updatable {...commonProps} />}
+        </>
+    );
+}
+
+function Updater() {
+    const settings = useSettings(["notifyAboutUpdates", "autoUpdate", "autoUpdateNotification"]);
+    const updaterContent = updaterBody();
```

Comment:
React components should always be created via jsx, not by calling it as a function:
```suggestion
    const updaterContent = <UpdaterBody />;
```